email/Check-in.html

```diff
@@ -0,0 +1,46 @@
+
+<html >
+	<head>
+	<title>Notification</title>
+	<link rel="stylesheet" type="text/css" href="stylesheets/email.css" />
```

Comment:
These templates will be used on backend, so to prevent problems with file paths - styles should be embedded to html